src/components/NavigationBar/NavigationBar.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { NavigationBar } from "./NavigationBar";
+import { create } from "react-test-renderer";
+import { shallow } from "enzyme";
+
+describe("<NavigationBar/>", () => {
+  it("renders the snapshot correctly", () => {
+    const component = create(
+      <NavigationBar
+        targets={[
+          {
+            name: "home",
+            url: "/",
+          },
+        ]}
+      />
+    );
+
+    expect(component).toMatchInlineSnapshot(`
+      <div
+        className="navigationBar-container"
+      >
+        <ul>
+          <nav>
+            <li
+              className="text-piped"
+              data-test="navTarget-name"
+            >
+              <a
+                className="govuk-link lbh-link"
+                href="//"
+              >
+                home
+              </a>
+            </li>
+          </nav>
+        </ul>
+      </div>
+    `);
+  });
+
+  it("renders the correct number of nav links", () => {
```

Comment:
Just from the snapshot listed above, it might be worth adding a test that checks the href property of the `<a>` tags.

Also, I noticed that the stories and tests are all around slugs and not full URLs. There's a chance that this nav-bar will have to point at other domains (e.g. single view). Would be good to get some tests in around that too.